src/control/lib/control/mocks.go

```diff
@@ -84,6 +84,8 @@ func (mi *MockInvoker) InvokeUnaryRPC(_ context.Context, uReq UnaryRequest) (*Un
 		uResp.Responses = addMSResp(&mgmtpb.PoolCreateResp{})
 	case *PoolDestroyReq:
 		uResp.Responses = addMSResp(&mgmtpb.PoolDestroyResp{})
+	case *ContSetOwnerReq:
+		uResp.Responses = addMSResp(&mgmtpb.ContSetOwnerResp{})
```

Comment:
This is the right place for this until #2430 lands (just checked, looks like it's finished with CI), and then it will be moved over into `cmd/dmg/command_test.go`. It was a mistake to put this in the control package, as it's only needed to make the dmg testing scenarios work correctly.

Reply:
Resolved it with the merge.